Food/bee.lua

```diff
@@ -0,0 +1,50 @@
+local name = "bee"
```

Comment:
Would you mind removing this file? Let's keep this separate in #104

Reply:
Sorry i messed up my master branch, it is fixed.